packages/decentraland-ecs/src/setupUtils.ts

```diff
@@ -42,33 +55,66 @@ export function entityV3FromFolder({ folder, addOriginalPath }: { folder: string
         }
         const _folder = folder.replace(/\\/gi, '/')
         const key = file.replace(_folder, '').replace(/^\/+/, '')
+        return key
+      })
+      .filter(($) => !!$) as string[]
+
+    const ig = ignore().add(ignorePattern || '')
```

Comment:
Shouldn't this also include node_modules and .git? Or that should come from .dclignore?

Reply:
Maybe consider that as default if no .dclignore file is provided or is empty?

Reply:
Yep, it's a good idea. **.dclignore** is generated when you create a new project (https://github.dev/decentraland/cli/blob/a90a99e0060f82b864b14b283e4ba9552509488b/src/lib/Project.ts) but it can be deleted accidentally